.github/workflows/ci.yml

```diff
@@ -22,10 +22,29 @@ jobs:
 
     - name: .NET Restore
       run: dotnet restore
+
     - name: .NET Build
       run: dotnet build --no-restore
+
     - name: .NET Test
       run: dotnet test --no-build --verbosity normal
 
     - name: Docker Build
       run: docker build . -f ./KeyVaultCA.Web/Dockerfile -t keyvault-ca-web:$(date +%s)
+
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v1
+
+    - name: Terraform Init
+      run: terraform init -input=false
+
+    - name: Terraform Format
```

Comment:
Can you also add a plan step? That would ensure that the Terraform content is fully valid.

Reply:
Plan step unfortunately requires authentication to Terraform. Should we do it already or as part of the CD pipeline?

Reply:
I see, then makes sense to leave it for the CD.